eng/versioning/version_client.txt

```diff
@@ -11,7 +11,7 @@ com.azure:azure-core-amqp;1.1.2;1.2.0-beta.1
 com.azure:azure-core-http-jdk-httpclient;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-core-http-netty;1.5.1;1.6.0-beta.1
 com.azure:azure-core-http-okhttp;1.2.2;1.3.0-beta.1
-com.azure:azure-core-management;1.0.0-beta.8;1.0.0-beta.8
+com.azure:azure-management-core;1.0.0-beta.8;1.0.0-beta.8
```

Comment:
/CC @JonathanGiles 
I'm curious. com.azure:azure-core-management has previously been released several times, is there any reason why the artifact ID is being changed?

Reply:
thanks @JimSuplizio for reviewing. Actually this is based on @JonathanGiles feedback.

Reply:
/CC @mitchdenny 
Everything under sdk/core is azure-core-<something>, and their directories match the artifact name. If this is being renamed then there probably should be more changes than just renaming the artifact. @JonathanGiles is this no longer a core artifact, is this supposed to moving somewhere else or is there a different naming convention happening for management?

Reply:
Thanks @JimSuplizio . Personal impression on current package: no lib other than mgmt SDK would benefit for including the azure-core-management lib. So this probably no very suitable for a core lib.

Yes, there are likely follow-up actions if we do change artifactId. For now this part is only trial on a suggestion. I won't merge the artifactId change without approval from you or Jonathon.